code/mod_table/mod_table.cpp

```diff
@@ -790,6 +790,29 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&SCPUI_loads_hi_res_animations);
 		}
 
+		if (optional_string("$Far clip plane:")) {
```

Comment:
I think naming these parameter "Max draw distance" and "Min draw distance" is clearer and allows us to get away with not having to explain the concept of a clip plane to users